Ironmon-Tracker.lua

```diff
@@ -1,13 +1,21 @@
--- This file is the first file loaded by Bizhawk.
+-- This file is the first file loaded by Bizhawk or mGBA.
 -- This file is NOT automatically updated live during Tracker auto-updates; requires Bizhawk restart
```

Comment:
Yes still true. This is because there are two ways to update. They most common is still the one people use now, where they load the real tracker script into Bizhawk, which cannot be unloaded